website/docs/r/dl_gateway_action.html.markdown

```diff
@@ -0,0 +1,193 @@
+
+subcategory: "Direct Link Gateway"
+layout: "ibm"
+page_title: "IBM : dl_gateway_action"
+description: |-
+  Manages IBM Direct Link Gateway.
+---
+
+# ibm_dl_gateway_action
+
+This resource used for provider created Direct Link Connect gateways to approve or reject specific changes initiated from a provider portal For more information, see [about Direct Link](https://cloud.ibm.com/docs/dl?topic=dl-dl-about).
+
+
+## Sample usage to approve the provider created Direct Link of connect type
+In the following example, you can approve the provider created Direct Link of connect type:
+
+
+---
+```terraform
+data "ibm_dl_provider_ports" "test_ds_dl_ports" {
+}
+resource ibm_dl_provider_gateway "test_dl_gateway" {
+	bgp_asn =  64999
+	name = "example-Gateway"
+	speed_mbps = 1000 
+	port = data.ibm_dl_provider_ports.test_ds_dl_ports.ports[0].port_id
+	customer_account_id = "3b1bd7fa2bc3406ea70ba4ade8aa3f1b"			  		                  
+}
+data "ibm_dl_gateway" "test_ibm_dl_gateway" {
+	name = ibm_dl_provider_gateway.test_dl_gateway.name		
+}
+resource ibm_dl_gateway_action "test_dl_gateway_action" {
```

Comment:
The quotes are missing here "ibm_dl_gateway_action"

Reply:
Fixed